DEVX-1524: Adds a protobuf example configuration (#61)
confluentinc/kafka-connect-datagen: README.md

See the [config](https://github.com/confluentinc/kafka-connect-datagen/tree/master/config) folder for sample configurations. 

## Supported data formats

> Kafka Connect supports [Converters](https://docs.confluent.io/current/connect/userguide.html#connect-configuring-converters) which can be used to convert record key and value formats when reading from and writing to Kafka. As of the 5.5 release, Confluent Platform shipped JSON and Protobuf converters to go along with the existing Avro converter.

Reviewer: ```suggestion
Kafka Connect supports [Converters](https://docs.confluent.io/current/connect/userguide.html#connect-configuring-converters) which can be used to convert record key and value formats when reading from and writing to Kafka. As of the 5.5 release, Confluent Platform packages Avro, JSON, and Protobuf converters (earlier versions package just Avro converters).
```

> For an example of using the the Protobuf converter with kafka-connect-datagen, see this [example configuration](config/connector_users_protobuf.config). Take note of the required use of the `SetSchemaMetadata` [Transformation](https://docs.confluent.io/current/connect/transforms/index.html) to address a known issue with compatibility of schema names used by kafka-connect-datagen and Protobuf.

Reviewer: ```suggestion
For an example of using the the Protobuf converter with kafka-connect-datagen, see this [example configuration](config/connector_users_protobuf.config). Note the use of the `SetSchemaMetadata` [Transformation](https://docs.confluent.io/current/connect/transforms/index.html) which addresses a compatibility issue between schema names used by kafka-connect-datagen and Protobuf.
```

Reviewer: @rspurgeon please elaborate on the issue -- just a 5 word summary.  

FYC also submit a GH issue that captures the issue in more detail and link here
